src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp

```diff
@@ -368,6 +371,8 @@ HOSTFXR_UTILITY::GetAbsolutePathToDotnet(
     // Only do it if no path is specified
     if (requestedPath.has_parent_path())
     {
+        WLOG_INFOF(L"Absolute path to dotnet.exe was not found at %s", requestedPath.c_str());
```

Comment:
It's not "not found", it just wasn't an absolute path.

Reply:
So what would we have in event log in this case?

Reply:
L"Invalid or unknown processPath provided in web.config: processPath = '%s', ErrorCode = '0x%x'."